CONTRIBUTING.adoc

```diff
@@ -65,17 +66,22 @@ $ brew install git
 ----
 
 Then, follow the instructions to install https://docs.docker.com/docker-for-mac/install/[`docker`].
+Search the https://docs.docker.com/docker-for-mac/release-notes/[Docker for Mac releases] to find a compatible
+version to download.
 You will need to build `linux/amd64` binaries for the OpenShift server; if you
 want to do the builds locally, you will need to follow the instructions to
 install https://golang.org/doc/install#install[the `go` programming language].
+Go `pkg` installers can be downloaded https://golang.org/dl/[here].
```

Comment:
go is also available via brew

Reply:
Current version of go in Homebrew is 1.10 - installing `go@1.9` is keg-only, which means you have to manually set up your GOROOT. This gave me enough of a headache that I decided to go with the pkg installer instead.